cmd/nerdlog/options.go

```diff
@@ -95,11 +98,31 @@ var AllOptions = map[string]*OptionMeta{
 			o.MaxNumLines = maxNumLines
 			return nil
 		},
-		Help: "How many log lines to fetch from each logstream in one query",
+		Help: "How many log messages to fetch from each logstream in one query",
 	},
 	"numlines": {
 		AliasOf: "maxnumlines",
 	}, // }}}
+	"ephemeralkeyproviderenabled": {
+		Get: func(o *Options) string {
+			if o.EphemeralKeyProviderEnabled {
+				return "true"
+			}
+			return "false"
+		},
+		Set: func(o *Options, value string) error {
+			switch value {
+			case "true", "1", "yes", "on":
+				o.EphemeralKeyProviderEnabled = true
+			case "false", "0", "no", "off":
+				o.EphemeralKeyProviderEnabled = false
+			default:
+				return errors.Errorf("invalid value for ephemeralkeyproviderenabled: %s", value)
+			}
+			return nil
+		},
+		Help: "Enable or disable ephemeral SSH key support",
```

Comment:
So the PR adds this option `EphemeralKeyProviderEnabled`, and goes as far as parsing all these `"true", "1", "yes", "on"`, but then never uses it. :thinking: 

I don't think we need this option in the first place.